### ISSUE TITLE
Add support for Sinopé TH1500ZB

### ISSUE THREAD

**Issue by MattL0** [dresden-elektronik/deconz-rest-plugin#5848]
## Device

- Product name:  Sinopé TH1500ZB Smart double-pole thermostat
- Manufacturer: Sinope Techonologies.
- Model identifier: TH1500ZB.
- Device type : Please remove all unrelated device types. 
  - Thermostat
 

## Screenshots
![image](https://user-images.githubusercontent.com/27497561/157180074-eb737834-5de0-40bd-9dae-6916f28ada2d.png)
![image](https://user-images.githubusercontent.com/27497561/157180131-25c98ad4-256c-4635-a981-cd172ab0f9bd.png)
![image](https://user-images.githubusercontent.com/27497561/157180203-4e1b1550-247f-4873-91e1-43f9438342e8.png)
![image](https://user-images.githubusercontent.com/27497561/157180346-e14a2f20-9845-4786-b346-3a20580a640a.png)
![image](https://user-images.githubusercontent.com/27497561/157180435-1bbea3a2-e426-43a0-97e7-c44b993a8cb2.png)
![image](https://user-images.githubusercontent.com/27497561/157180480-b504bb62-ea60-47de-8e3e-e4bcc0bcac94.png)
![image](https://user-images.githubusercontent.com/27497561/157180597-e48082bb-fd5c-4fdc-9fbe-7137fb8f1d85.png)
![image](https://user-images.githubusercontent.com/27497561/157180751-4713dc7c-87fc-4afa-ace3-6d58c2ba96ec.png)
![image](https://user-images.githubusercontent.com/27497561/157180841-edc169b3-af5e-4b69-9b1d-aced60876c12.png)
![image](https://user-images.githubusercontent.com/27497561/157180987-52960016-88cb-4681-ae77-8dd38a1edc88.png)
![image](https://user-images.githubusercontent.com/27497561/157181050-b99a1cf0-54ae-4c27-9e27-4a20a9da2871.png)
![image](https://user-images.githubusercontent.com/27497561/157181102-8e8bc98a-f3a9-49d6-ae88-bd370126798e.png)
![image](https://user-images.githubusercontent.com/27497561/157181153-0186fad5-4ff9-49f3-af9d-21909c7cd9de.png)
![image](https://user-images.githubusercontent.com/27497561/157181211-b0792179-2831-4b32-80a8-f6266702f999.png)
![image](https://user-images.githubusercontent.com/27497561/157181267-03042e99-7d3d-4226-be1e-b5898e98fa59.png)
![image](https://user-images.githubusercontent.com/27497561/157181342-a7d9a0bd-0e79-4f64-b0d4-dcc0598425d9.png)
![image](https://user-images.githubusercontent.com/27497561/157181386-57fd75e3-e911-4747-a10f-8382c391bb86.png)
![image](https://user-images.githubusercontent.com/27497561/157181454-3e61242d-7271-4740-be71-8c11c65007d1.png)
![image](https://user-images.githubusercontent.com/27497561/157181503-29b2a325-b59a-4838-a521-9625189b03b4.png)
![image](https://user-images.githubusercontent.com/27497561/157181569-85c25141-2887-4c4f-b629-3d471b2a7dc8.png)
![image](https://user-images.githubusercontent.com/27497561/157181610-31b7856b-694e-435d-abec-f5725af676f4.png)
![image](https://user-images.githubusercontent.com/27497561/157181680-e9eebc14-3094-4a13-b97d-d348f52f3dfd.png)
![image](https://user-images.githubusercontent.com/27497561/157181750-eb0731b5-2e07-401e-a33d-3df05e2f78c2.png)
![image](https://user-images.githubusercontent.com/27497561/157181838-a62b7d91-15aa-4614-bc42-78153cf653b3.png)
![image](https://user-images.githubusercontent.com/27497561/157181929-64044c19-b7a2-40dd-87e2-224cbdf26a19.png)

**Comment by MattL0** [comment]
I see nothing from the api (nodered, deconz phoscon)

**Comment by Smanar** [comment]
Hello, I m searching if it's possible to add thermostat using DDF, as your device look standard.
But I don't see one except this one but still in draft status https://github.com/dresden-elektronik/deconz-rest-plugin/pull/5793

**Comment by BabaIsYou** [comment]
It looks similar to others Sinopé thermostats partly supported like TH1124ZB (https://github.com/dresden-elektronik/deconz-rest-plugin/blob/5ce086600955a6bd696a7b74ae1ac1bcb2f0c767/general.xml) but perhaps it has to be specifically whitelisted somewhere to be seesn uin deConz app.

**Comment by Smanar** [comment]
Ha nice, thx @BabaIsYou and this DDF is working ?

```
{
  "schema": "devcap1.schema.json",
  "manufacturername": "Sinope",
  "modelid": "TH1124ZB",
  "product": "TH1124ZB",
  "sleeper": false,
  "status": "Draft",
  "subdevices": [
    {
      "type": "$TYPE_THERMOSTAT",
      "restapi": "/sensors",
      "uuid": [
        "$address.ext",
        "0x01",
        "0x0201"
      ],
      "fingerprint": {
        "profile": "0x0104",
        "device": "0x0301",
        "endpoint": "0x01",
        "in": [
          "0x0000",
          "0x0201",
          "0x0204",
          "0x0B05"
        ]
      },
      "items": [
        {
          "name": "attr/id"
        },
        {
          "name": "attr/lastannounced"
        },
        {
          "name": "attr/lastseen"
        },
        {
          "name": "attr/manufacturername"
        },
        {
          "name": "attr/modelid"
        },
        {
          "name": "attr/name"
        },
        {
          "name": "attr/swversion"
        },
        {
          "name": "attr/type"
        },
        {
          "name": "attr/uniqueid"
        },
        {
          "name": "config/heatsetpoint",
          "description": "Target temperature of a thermostat.",
          "default": 0
        },
        {
          "name": "config/mode",
          "description": "Mode of the device."
        },
        {
          "name": "config/offset",
          "description": "Relative offset to the main measured value.",
          "default": 0
        },
        {
          "name": "config/on"
        },
        {
          "name": "config/reachable"
        },
        {
          "name": "config/schedule",
          "description": "On-device heating schedules."
        },
        {
          "name": "config/schedule_on",
          "description": "Determines if on-device schedules for setting the heatsetpoint are currently used or if the thermostat is operated manually."
        },
        {
          "name": "state/lastupdated"
        },
        {
          "name": "state/on",
          "description": "True when device is on; false when off.",
          "refresh.interval": 5
        },
        {
          "name": "state/temperature",
          "description": "The current temperature in °C &times; 100.",
          "default": 0
        }
      ]
    },
    {
      "type": "$TYPE_POWER_SENSOR",
      "restapi": "/sensors",
      "uuid": [
        "$address.ext",
        "0x01",
        "0x0b04"
      ],
      "fingerprint": {
        "profile": "0x0104",
        "device": "0x0301",
        "endpoint": "0x01",
        "in": [
          "0x0000",
          "0x0B04"
        ]
      },
      "items": [
        {
          "name": "attr/id"
        },
        {
          "name": "attr/lastannounced"
        },
        {
          "name": "attr/lastseen"
        },
        {
          "name": "attr/manufacturername"
        },
        {
          "name": "attr/modelid"
        },
        {
          "name": "attr/name"
        },
        {
          "name": "attr/swversion"
        },
        {
          "name": "attr/type"
        },
        {
          "name": "attr/uniqueid"
        },
        {
          "name": "config/on"
        },
        {
          "name": "config/reachable"
        },
        {
          "name": "state/current",
          "description": "The measured current (in mA).",
          "refresh.interval": 300
        },
        {
          "name": "state/lastupdated"
        },
        {
          "name": "state/power",
          "description": "The measured power (in W).",
          "refresh.interval": 300
        },
        {
          "name": "state/voltage",
          "description": "The measured voltage (in V).",
          "refresh.interval": 300
        }
      ]
    },
    {
      "type": "$TYPE_CONSUMPTION_SENSOR",
      "restapi": "/sensors",
      "uuid": [
        "$address.ext",
        "0x01",
        "0x0702"
      ],
      "fingerprint": {
        "profile": "0x0104",
        "device": "0x0301",
        "endpoint": "0x01",
        "in": [
          "0x0000",
          "0x0702"
        ]
      },
      "items": [
        {
          "name": "attr/id"
        },
        {
          "name": "attr/lastannounced"
        },
        {
          "name": "attr/lastseen"
        },
        {
          "name": "attr/manufacturername"
        },
        {
          "name": "attr/modelid"
        },
        {
          "name": "attr/name"
        },
        {
          "name": "attr/swversion"
        },
        {
          "name": "attr/type"
        },
        {
          "name": "attr/uniqueid"
        },
        {
          "name": "config/on"
        },
        {
          "name": "config/reachable"
        },
        {
          "name": "state/consumption",
          "description": "The measured consumption (in Wh).",
          "refresh.interval": 300
        },
        {
          "name": "state/lastupdated"
        },
        {
          "name": "state/power",
          "description": "The measured power (in W).",
          "refresh.interval": 300
        }
      ]
    },
    {
      "type": "$TYPE_TEMPERATURE_SENSOR",
      "restapi": "/sensors",
      "uuid": [
        "$address.ext",
        "0x01",
        "0x0402"
      ],
      "fingerprint": {
        "profile": "0x0104",
        "device": "0x0301",
        "endpoint": "0x01",
        "in": [
          "0x0000",
          "0x0402"
        ]
      },
      "items": [
        {
          "name": "attr/id"
        },
        {
          "name": "attr/lastannounced"
        },
        {
          "name": "attr/lastseen"
        },
        {
          "name": "attr/manufacturername"
        },
        {
          "name": "attr/modelid"
        },
        {
          "name": "attr/name"
        },
        {
          "name": "attr/swversion"
        },
        {
          "name": "attr/type"
        },
        {
          "name": "attr/uniqueid"
        },
        {
          "name": "config/offset",
          "default": 0
        },
        {
          "name": "config/on"
        },
        {
          "name": "config/reachable"
        },
        {
          "name": "state/lastupdated"
        },
        {
          "name": "state/temperature",
          "description": "The current temperature in °C &times; 100.",
          "default": 0
        }
      ]
    }
  ],
  "bindings": [
    {
      "bind": "unicast",
      "src.ep": 1,
      "dst.ep": 1,
      "cl": "0x0204"
    },
    {
      "bind": "unicast",
      "src.ep": 1,
      "dst.ep": 1,
      "cl": "0x0B04",
      "report": [
        {
          "at": "0x050B",
          "dt": "0x29",
          "min": 1,
          "max": 300,
          "change": "0x00000001"
        },
        {
          "at": "0x0505",
          "dt": "0x21",
          "min": 1,
          "max": 300,
          "change": "0x0000000A"
        },
        {
          "at": "0x0508",
          "dt": "0x21",
          "min": 1,
          "max": 300,
          "change": "0x00000001"
        }
      ]
    }
  ]
}
```

**Comment by MattL0** [comment]
Hi, I am very sorry for the lack of reply. I got a busy schedule. 

I had tu return the device to amazon, It was making a weird  and loud click noise when controlling the heater. Maybe a relay. 
I am not sur if this is an issue or not, I think I saw somewhere on the web somebody having the same issue, so I didn’t take the chance to pass the return delay windows.

I guess if the TH1123/1124 don’t have this issue, maybe I can retry the th1500 this summer.
